TrialToursBundle.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MauticPlugin\TrialToursBundle;
+
+use Mautic\PluginBundle\Bundle\PluginBundleBase;
+
+class TrialToursBundle extends PluginBundleBase
```

Comment:
I think the bundle name will have to be changed in more locations like here.

Reply:
Sorry, was too fast. I renamed the bundle everywhere to ProductTour.